product_packaging_type/models/product_packaging_type.py

```diff
@@ -57,7 +57,11 @@ def default_packaging_type_id(self):
         related="packaging_type_id.is_default",
         store=True,
     )
-    type_has_gtin = fields.Boolean(related="packaging_type_id.has_gtin", readonly=True)
+    type_has_gtin = fields.Boolean(readonly=True, compute="_compute_type_has_gtin")
```

Comment:
@Tonow-c2c this one must be removed

AttributeError: 'product.packaging' object has no attribute '_compute_type_has_gtin'


```suggestion
```

Reply:
thanks :pray:  sorry for the issue in the rebase :grimacing: